tools/castle-editor/code/framedesign.pas

```diff
@@ -2197,6 +2309,7 @@ procedure TDesignFrame.MenuItemViewportCameraViewAllClick(Sender: TObject);
     V.Navigation.ModelBox := Box;
 
   ModifiedOutsideObjectInspector;
+  RecordUndo('MenuItemViewportCameraViewAllClick'); //whatever that means?
```

Comment:
Should be `RecordUndo('Camera Current := View All');`

Reply:
Thanks, fixed!